docker/config/local_dev.env

```diff
@@ -101,6 +101,7 @@ RABBITMQ_VIRTUAL_HOST=rabbitvhost
 RAVEN_DSN=
 SECRET_KEY=secretkey
 STATSD_HOST=statsd
+TELEMETRY_BUCKET_NAME=telemetry_bucket
```

Comment:
Previously, the webapp blithely ignored the telemetry bucket. Now it needs to know, so I added a variable. We'll need to add this to the -stage, -prod, -new-stage, -new-prod, and other environments as well.